didman/didman.go

```diff
@@ -125,7 +133,11 @@ func (d *didman) GetCompoundServices(id did.DID) ([]did.Service, error) {
 }
 
 func (d *didman) AddCompoundService(id did.DID, serviceType string, endpoints map[string]ssi.URI) (*did.Service, error) {
-	log.Logger().Debugf("Adding compound service (did: %s, type: %s, endpoints: %v)", id.String(), serviceType, endpoints)
+	log.Logger().
+		WithField("did", id.String()).
+		WithField("serviceType", serviceType).
+		WithField("serviceEndpoint", fmt.Sprintf("%v", endpoints)).
```

Comment:
Doesn't the lib expand these slices automagically?

Reply:
Yes